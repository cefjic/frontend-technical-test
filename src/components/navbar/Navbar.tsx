import Link from "next/link";
import React from "react";
import { Wrapper, Title } from "./Navbar.styles";

const Navbar = () => {
  return (
    <Wrapper>
      <Link href="/" passHref>
        <Title>Messenger</Title>
      </Link>
    </Wrapper>
  );
};

export default Navbar;
