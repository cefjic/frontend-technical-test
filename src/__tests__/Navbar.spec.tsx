import { render } from "@testing-library/react";
import React from "react";
import Navbar from "../components/navbar/Navbar";

describe("Layout component", () => {
  it("should render a component", () => {
    const { container } = render(<Navbar />);

    expect(container).toMatchSnapshot();
  });
});
