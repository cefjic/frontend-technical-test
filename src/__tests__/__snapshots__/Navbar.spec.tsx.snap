// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should render a component 1`] = `
<div>
  <header
    class="Navbarstyles__Wrapper-sc-vhmymz-0 cGHFyD"
  >
    <h1
      class="Navbarstyles__Title-sc-vhmymz-1 fUIVQd"
      href="/"
    >
      Messenger
    </h1>
  </header>
</div>
`;
